src/ekomark/benchmark/external/apfel_utils.py

```diff
@@ -72,8 +72,8 @@ def compute_apfel_data(
 
     # Run
     apf_tabs = {}
-    for q2 in operators["mu2grid"]:
-        apfel.EvolveAPFEL(theory["Q0"], np.sqrt(q2))
+    for mu2 in np.array(operators["mugrid"]) ** 2:
+        apfel.EvolveAPFEL(theory["Q0"], np.sqrt(mu2))
```

Comment:
here you can avoid the square now ...

Reply:
yes but in

https://github.com/NNPDF/eko/blob/889b855ca818eb82e39bfc51799e8a7a25d8a884/src/ekomark/benchmark/external/apfel_utils.py#L114

the table is constructed using the squares. To either i loop over the squares and I use `sqrt` or I do the opposite.
I did in this way in order to change the minimum possible

Reply:
if you then square below ...